source/fa/1.0.0/index.html.haml

```diff
@@ -0,0 +1,250 @@
+
+.effort
+
+  %h3#effort
+    %a.anchor{ href: "#effort", aria_hidden: "true" }
+    چطور ثبت تغییرات پروژه را آسان‌تر کنیم؟
+
+  %p
+    در بالای فایل تاریخچهٔ تغییرات، بخشی را تحت عنوان <code>Unreleased</code> برای دنبال‌کردن تغییرات پیش‌رو در نظر بگیرید.
+
+  %p این کار دو فایده دارد:
+
+  %ul
+    %li
+      اولاً کاربران می‌توانند پیش از انتشار یک نسخهٔ جدید، از تغییراتی که می‌توانند در نسخهٔ بعدی انتظار داشته باشند مطلع شوند.
+    %li
+      دوماً هنگام انتشار نسخهٔ جدید، می‌توانید به‌راحتی بخش <code>Unreleased</code> را در قالب یک نسخهٔ جدید به بخش انتشاریافته‌ها منتقل کنید و دیگر لازم نباشد بروید ببینید چه تغییراتی ایجاد شده است.
+
+.bad-practices
+  %h3#bad-practices
+    %a.anchor{ href: "#bad-practices", aria_hidden: "true" }
+    آیا تاریخچهٔ تغییرات می‌تواند بد نوشته شود؟
+  
+  %p بله. در شرایط زیر.:
+  
+  %h4#log-diffs
+    %a.anchor{ href: "#log-diffs", aria_hidden: "true" }
+    استفاده از خروجی log diff به عنوان تاریخچهٔ تغییرات
+
+  %p
+  استفاده از خروجی log diffها به عنوان تاریخچهٔ تغییرات، ایدهٔ بدی است. این خروجی مملو از اطلاعات درهم‌وبرهم است؛ مثلاً کامیت‌های مربوط به مرج، کامیت‌هایی با عناوین مبهم، تغییرات مربوط به مستندات و غیره.
+
+  %p
+    هدف از یک کامیت، ثبت یک گام انجام‌شده در روند تکامل سورس کد است. برخی از پروژه‌ها، کامیت‌های تمیزی دارند و برخی نه.
+
+  %p
+  هدف از یک مدخل جدید در فایل تاریخچهٔ تغییرات، مستندسازی شفاف تغییرات مهمِ پروژه برای کاربران نهایی است. هر مدخل در تاریخچهٔ تغییرات لزوماً متناظر با یک کامیت در تاریخچهٔ کامیت‌ها نیست. ممکن است چندین کامیت مختلف، مربوط به یک تغییر باشد و بالعکس یک کامیت (غیراستاندارد) دربردارندهٔ چندین تغییر مهم باشد.
+
+  %h4#ignoring-deprecations
+    %a.anchor{ href: "#ignoring-deprecations", aria_hidden: "true" }
+    نادیده‌گرفتن چیزهای منسوخ‌شده
+    
+  %p
+    وقتی کاربران از یک نسخه به نسخهٔ دیگر به‌روزرسانی می‌کنند، باید بدانند که چه موقع نسخهٔ جدید، تغییرات ناسازگار به‌همراه دارد؛ یعنی مثلاً بعد از بروزرسانی یک کتابخانه که پروژهٔ شما به آن وابسته بوده، آیا بخش‌هایی از پروژه از کار می‌افتد یا نه. اگر بله، کدام بخش‌ها. پاسخ به این سؤال کمک می‌کند توسعه‌دهنده‌ها متناسب با نسخهٔ جدید، قسمت‌های منسوخ‌شده را جایگزین کرده و بعد از آن اقدام به بروزرسانی کنند.
+
+  %h4#confusing-dates
+    %a.anchor{ href: "#confusing-dates", aria_hidden: "true" }
+    تاریخ‌های گیج‌کننده
+
+  %p
+    در هر نقطه‌ای از دنیا شکل نمایش تاریخ متفاوت است؛ بنابراین پیداکردن یک قالب نمایش همه‌فهم برای کل مردم دنیا کار دشواری است. مزیت استفاده از قالب‌هایی مثل <code>17-07-2017</code> این است که در آن ترتیب بزرگترین به کوچکترین واحدها یعنی سال، ماه و روز رعایت شده است. این باعث می‌شود که برخلاف برخی از قالب‌ها که در آن جای ماه و روز عوض شده، درک تاریخ دچار ابهام نشود. با درنظرگرفتن این دلایل و این واقعیت که این فرمت، یک #{link_to "استاندارد ایزو", data.links.isodate} است، این قالب را برای مدخل‌های تاریخچهٔ تغییرات در نظر گرفته‌ایم.
+  
+  %aside
+    اگر فکر می‌کنید جای موارد بیشتری در این لیست خالی است با
+    = link_to "ارسال ایشو", data.links.issue
+    یا ارسال پول‌ریکوئست به من کمک کنید.
```

Comment:
Is it weird that this gets translated to "or send money request help me."? 😆 

I wouldn't want people to think we're asking for money but maybe it's technical jargon and Google Translate is stumbling on it?

Reply:
Oh man, that killed me!

the phrase پول‌ریکوئست doesn't mean "money request" 😂 
In fact, It hasn't been translated at all. It is just equivalent to American pronunciation of this phrase which is written in Persian alphabets. the "pull" is pronounced like پول and the "request" like ریکوئست, so combining these two, results پول‌ریکوئست.

**Why didn't I translated it?**
Because there is still no acceptable Persian translation for this technical term.

**Why didn't I leave it not translated?**
Because using English words in Persian text doesn't look good and makes it harder to read, so it is preferable to use Persian words as much as possible.

**Why didn't I do that for all terms?**
Because it is unusual or difficult to understand some English words that written in Persian alphabets, so I choose to leave them alone. (or maybe I've forgotten it)

By the way, don't worry @olivierlacan, I don't translate the text in the way that the meaning is changed, at least, not on purpose!

I think It took me more longer to explain this than sending this pull request (پول‌ریکوست). 🤦‍♂️

Reply:
Hah! I really appreciate you taking the time to explain though, that’s super nice of you! 

I’ll try to see if we can vendor that font, shouldn’t be too hard if their license allows it. 😃